src/main/webapp/app/course/plagiarism-cases/instructor-view/detail-view/plagiarism-case-instructor-detail-view.component.ts

```diff
@@ -0,0 +1,130 @@
+                this.metisService.setPageType(this.pageType);
+                this.metisService.getFilteredPosts({
+                    plagiarismCaseId: this.plagiarismCase!.id,
+                });
+                this.createEmptyPost();
+            },
+        });
+        this.postsSubscription = this.metisService.posts.pipe().subscribe((posts: Post[]) => {
+            this.posts = posts;
+        });
+    }
+
+    ngOnDestroy(): void {
+        this.postsSubscription?.unsubscribe();
+    }
+
+    /**
+     * saves the verdict of the plagiarism case as POINT_DEDUCTION
+     * and saves the point deduction in percent
+     */
+    savePointDeductionVerdict(): void {
+        this.plagiarismCasesService
+            .savePlagiarismCaseVerdict(this.courseId, this.plagiarismCaseId, {
+                verdict: PlagiarismVerdict.POINT_DEDUCTION,
+                verdictPointDeduction: this.verdictPointDeduction,
+            })
+            .subscribe({
+                next: (res: HttpResponse<PlagiarismCase>) => {
+                    this.plagiarismCase.verdict = res.body!.verdict;
+                    this.plagiarismCase.verdictPointDeduction = res.body!.verdictPointDeduction!;
+                },
+            });
+    }
+
+    /**
+     * saves the verdict of the plagiarism case as WARNING
+     * and saves the warning message
+     */
+    saveWarningVerdict(): void {
+        this.plagiarismCasesService
+            .savePlagiarismCaseVerdict(this.courseId, this.plagiarismCaseId, {
+                verdict: PlagiarismVerdict.WARNING,
+                verdictMessage: this.verdictMessage,
+            })
+            .subscribe({
+                next: (res: HttpResponse<PlagiarismCase>) => {
+                    this.plagiarismCase.verdict = res.body!.verdict;
+                    this.plagiarismCase.verdictMessage = res.body!.verdictMessage!;
+                },
+            });
+    }
+
+    /**
+     * saves the verdict of the plagiarism case as PLAGIARISM
+     */
+    saveVerdict(): void {
+        this.plagiarismCasesService.savePlagiarismCaseVerdict(this.courseId, this.plagiarismCaseId, { verdict: PlagiarismVerdict.PLAGIARISM }).subscribe({
+            next: (res: HttpResponse<PlagiarismCase>) => {
+                this.plagiarismCase.verdict = res.body!.verdict;
+            },
+        });
+    }
+
+    /**
+     * invoke metis service to create an empty default post that is needed on initialization of a modal to create a post,
+     * this empty post has no course-wide context as well as the plagiarism case set as context
+     * it has an example text for the instructor and a default title containing the exercise title
+     **/
+    createEmptyPost(): void {
+        this.createdPost = this.metisService.createEmptyPostForContext(undefined, undefined, undefined, this.plagiarismCase);
+        this.createdPost.content = `Dear ${this.plagiarismCase.student!.name},\nAfter a meticulous review of your final submission for the ${
```

Comment:
Is it possible to move this string to language json files and make it translatable?

Reply:
Well, yes. But I'm unsure about how much sense this would make. This is only the suggestion for the instructor of how to write the notification for the student. In most cases the student would probably receive both versions (english and german) with one of them as an answer post. Also, it is planned to extend this functionality in the future to allow the instructors to set a course-wide default message, but for now I think this is fine (to keep this PR at least a little bit smaller). Let me know what you think 😃